### DISCUSSION TITLE
<title>gnore this, its a demonstration for a Youtube Video #2

### DISCUSSION THREAD

**Discussion by nxtshield** [community/community#106621] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Product Feedback

### Body

H#

**Comment by nxtshield** [comment]
Response #2

**Reply by ghost** [reply]
H3

**Comment by ghost** [comment]
HH#

**Comment by queenofcorgis** [comment]
We made the decision to disable the ability to earn Achievements in our Community in order to discourage users from participating in coordinated or inauthentic activity like rapid questions and answers in order to earn badges. You can learn more about this decision in our announcement post here [Achievements will no longer be available in the Community](https://github.com/orgs/community/discussions/106536).

Note that GitHub's [Acceptable Use Policies](https://docs.github.com/en/site-policy/acceptable-use-policies/github-acceptable-use-policies) prohibits coordinated or inauthentic activity like rapid questions and answers. As a result, we'll be unmarking the answer and locking this post. 

Any future violations may result in a temporary or indefinite block from the Community. Thanks for understanding.
